File: function.js
var number = prompt('Digite um número inteiro: ');

var sum = 0;

for(var i = 3; i < number; i++){
    if(i % 3 == 0 || i % 5 == 0){
        sum += i;
    }
}

console.log(`A soma resultante é: ${sum}`)

File: primeira_etapa/function.dart
import 'dart:io';

void main() {
  print('Digite um número inteiro:');
  String number = stdin.readLineSync() ?? "0";

  num value = int.parse(number);

  int sum = 0;

  for (var i = 3; i < value; i++) {
    if (i % 3 == 0 || i % 5 == 0) {
      sum += i;
    }
  }

  print("A soma resultante é: $sum");
}
